website/source/docs/builders/azure-setup.html.md

```diff
@@ -123,7 +123,7 @@ $ azure account show --json | jq -r ".[] | .id"
 ```
 
 Python:
-```shell
+``` shell
```

Comment:
I would have said no space, but I reviewed the file and we are inconsistent in the usage.  I am not sure what the official stance is. 😄

Reply:
I've always gone with no space :) I wasn't sure whether this fixed the formatting issue or not, but either way, I couldn't see anything else which would cause the formatting issue I am seeing on the live site, so I figured I'd at least make it consistent!